packages/utils/src/array.js

```diff
@@ -33,3 +33,5 @@ export const max = array =>
   Array.isArray(array) && array.length > 0
     ? array.reduce((result, value) => Math.max(value, result), Number.NEGATIVE_INFINITY)
     : undefined;
+
+export const toArray = input => (Array.isArray(input) ? input : [input]);
```

Comment:
I would be also keen to name this just `array`, but I went with the "safest" approach

Reply:
I prefer `toArray` :+1: